### ISSUE TITLE
Media: Update the Naming Convention Logic Follow For All Future Instance Bins and Ecobins

### ISSUE THREAD

**Issue by zcolah** [zesty-io/manager-ui#2499]
# Problem

1. Currently when an instance is made, it's instance bin / ecobin gets the word zesty at the start.
2. For example, if I have an instance called "UX Library', it gets the name `zesty-ux-library`
3. It is not clear to the user that this is the parent EcoBin/InstanceBin

#S
Potentially we set the default naming convention to one of the following:

1. UX Library Ecobin
2. UX Library Drive

I feel it reads better and the word drive is a word they can relate to it more from one drive google drive.

**Comment by giseleblair** [comment]
I like the word "Drive" to refer to an instance's media bin/bucket. 
The interface still displays these as "Folders" which can be confusing. And even the icon itself being a folder can be confusing to the user. 
![Screenshot 2024-02-28 at 10 49 35 AM](https://github.com/zesty-io/manager-ui/assets/9896546/4abe0d21-c1b9-41e5-9e8b-777bda021afb)

The process of creating a subfolder can be improved as well. The user should first select the "Drive" then the Folder they would like to add the subfolder to. 

![Screenshot 2024-02-28 at 10 51 07 AM](https://github.com/zesty-io/manager-ui/assets/9896546/51b9e6cf-8093-4684-958d-fa5440d08682)

**Comment by toddrsabo** [comment]
I agree with all points.  Drive is the right naming convention and we need a better icon to represent it other than a folder.

**Comment by zcolah** [comment]
Thank you for sharing your feedback here @giseleblair @toddrsabo @agalin920 

1. @agalin920 Which service/repo is responsible for this? 
2. @giseleblair I believe @agalin920 brings up a valid point that Bin is what our API docs call it at the moment. What is your opinion of us transitioning to the Word drive instead of Bin across the board (including our APIs and API docs)? Are there any drawbacks to that?

**Comment by giseleblair** [comment]
@zcolah  no concern with renaming this in the UI. There are inevitably going to be some inconsistencies between naming conventions in the API vs the UI as we have now. We still need to continue to improve on our design and communication without letting the API hold us back
